drivers/spi/spi_nrfx_spis.c

```diff
@@ -175,6 +177,10 @@ static int transceive(const struct device *dev,
 	const struct spi_buf *rx_buf = rx_bufs ? rx_bufs->buffers : NULL;
 	int error;
 
+#if defined(CONFIG_PM_DEVICE_RUNTIME)
+	pm_device_runtime_get(dev);
+#endif
```

Comment:
these calls have stubs when not enabled, so ifdeffery can go

Reply:
Done :)